### PR TITLE
Domupdates2

### PR DESCRIPTION
**Pull request by vladd-png**
### What's this PR Do?
Create a DOMUpdates.js file
- move domupdates from the index.js file into its respective class file for manager, user, and frontdesk
- add testing for dom updates with chai spies
### Where Should the Reviewer Start?
user, manager, and frontdesk test files as well as domupdates

### CODE REVIEW

**Comment by vladd-png** [comment]
@hannahhch @ryan-novak

File: src/DOMupdates.js [hunk 1]
```diff
@@ -1,3 +1,114 @@
...
-  
+import $ from 'jquery';
+
+let domUpdates = {
+  showError() {
+    $('#error-message').removeClass('hidden');
+  },
+  // --------------------------- Manager Functionality -----------------
+  loadManagerPage() {
+    $('#login-page').addClass('hidden').removeClass('visible');
```
Comment (ghost) [inline src/DOMupdates.js:9]: really like the double functionality of adding and removing a class simultaneously

File: src/classes/Manager.js [hunk 2]
```diff
@@ -1,12 +1,16 @@
-// import { } from '../index.js'
-// import User from './User.js'
+import domUpdates from '../DOMupdates.js';
+import User from './User';
 
-class Manager {
+
+class Manager extends User {
```
Comment (ghost) [inline src/classes/Manager.js:5]: Great job extending the user!

File: test/Manager-test.js [hunk 3]
```diff
@@ -1,18 +1,18 @@
 import chai from 'chai';
 const expect = chai.expect;
-// const spies = require('chai-spies');
-// chai.use(spies);
-
-// chai.spy.on(frontdesk, [totalDaysRevenue, findFullRooms], () => {});
-
+const spies = require('chai-spies');
```
Comment (ghost) [inline test/Manager-test.js:3]: Nice use of spies 🕵️‍♂️

**Comment by ghost** [comment]
Looks awesome!! Nice job breaking out all your DOM manipulation into that separate file and inheriting your User class when you instantiate your Manager

**Pull request merged**